app/posts/personalised-prevention-platform/2025/03/2025-03-31-technical-research.md

```diff
@@ -0,0 +1,120 @@
+
+Our current problem statement will continue to evolve over the course of discovery and alpha as we learn more about the problem we are trying to understand, and the hypotheses we are aiming to prove or disprove.
+
+## Purpose
+
+This analysis aims to give a simplified view of what NHS services, platforms and data could be available to technical solutions in the personalised prevention space.
+
+We explored 4 overarching hypotheses in the Personalised Prevention Platform Discovery  However, for the purposes of technology research, we focused on these 2 hypotheses:
+1. There is a serviceable cohort of people who will proactively manage their health to reduce their need for future treatment.
+2. We will be able to reach people early enough before health issues become problematic.
+
+This analysis is aimed at Product and Service colleagues in NHS England so they can visualise and understand how the available technology and data can best be used to feed into design considerations.
+
+## Background
+
+In the context of this early‑stage technology research, personalised prevention can be prompted by proactive campaigns for cohorts of people with shared characteristics, clinical conditions and risk factors.
+
+## What we explored
+
+### Understanding more about how to create people cohorts with shared characteristics and risk factors
+
+We wanted to understand what NHS England:
+- technology we can use to create, manage and enable the cohorting of people with shared characteristics
+- data sets and collections we can use to create cohorts of people
+- technology we can use to communicate with cohorts of people
+
+## What we did
+
+### Research the relevant NHS technology and data landscape
+
+Our research included several NHS internal technology platforms and data sets. The main ones reviewed throughout this research are:
+1. NHS Patient Demographics Service (PDS)
+2. NHS Cohorting as a Service (CaaS)
+3. NHS GP Extraction Service (GPES)
+4. NHS Hospital Episode Statistics (HES)
+5. NHS Notify
+6. NHS App
+
+#### What NHS England technology we can use to create, manage and enable the cohorting of people with shared characteristics
+
+The conclusion is that we can use the NHS Cohorting as a Service (CaaS) platform to create and manage cohorts of people for national campaigns.
+
+#### What NHS England data sets and collections we can use to create cohorts of people
+
+CaaS currently has access to several data services, sets and collections that can allow the creation of patient cohorts:
+- Personal Demographics Service (PDS)
+- General Practice Extraction Service (GPES)
+- Hospital Episodes Statistics (HES)
+- Systemic Anti-Cancer Therapy (SACT)
+- Radiotherapy Treatment Data Set (RTDS)
+
+#### What NHS England technology we can use to communicate with cohorts of people
+
+The conclusion is that we can use the NHS Notify platform to communicate with cohorts of people for national campaigns. This is because:
+
```

Comment:
Is this an unordered list? (Might well not be, just asking because of the lead-in)

Reply:
👍🏻